Projects/hGraphQL/Sources/Models/ContractModels.swift

```diff
@@ -135,6 +139,7 @@ public struct Contract: Codable, Hashable, Equatable {
         contract: GiraffeGraphQL.ContractsQuery.Data.Contract
     ) {
         id = contract.id
+        typeOfContract = TypeOfContract(rawValue: contract.typeOfContract.rawValue)!
```

Comment:
if you make typeOfContract optional we don't need ! 
or 
if it stays required you need to place " ?? .unknown" instead !

Reply:
Yeah, crossed my mind actually adding .unknown sounds good.